src/ensembl/src/content/app/browser/BrowserCog.tsx

```diff
@@ -25,6 +26,12 @@ const BrowserCog: FunctionComponent<BrowserCogProps> = (
   const toggleCog = useCallback(() => {
     if (cogActivated === false) {
       updateSelectedCog(index);
+
+      analyticsTracking.trackEvent({
+        category: 'track_settings',
+        label: index,
+        action: 'opened'
+      });
```

Comment:
Is this just a ui action of opening/closing the track config thingy? Do we need to track this event? I couldn't find anything about it in description or Jira tickets. @ens-st3 ?

Reply:
We should definitely track the opening. This is in the google doc (Action:Open) and I've added a specific question that would use it. Even though we decided that it can be cIosed by clicking anywhere, not just on the cog icon, I do think we should check that (Action:Close). I can't see a way that we can effectively track the final list of settings that were changed and I don't think it's very useful to do so